lte/gateway/python/magma/enodebd/enodeb_status.py

```diff
@@ -513,80 +553,82 @@ def _get_gps_status_as_bool(enodeb: EnodebAcsStateMachine) -> bool:
 
 
 def _get_and_cache_gps_coords(enodeb: EnodebAcsStateMachine) -> Tuple[
-    str, str,
+    str, str, str,
 ]:
     """
     Read the GPS coordinates of the enB from its configuration or the
     cached coordinate file if the preceding read fails. If reading from
     enB configuration succeeds, this method will cache the new coordinates.
 
     Returns:
-        (str, str): GPS latitude, GPS longitude
+        (str, str, str): GPS latitude, GPS longitude
     """
-    lat, lon = '', ''
+    lat, lon, ait = '', '', ''
     try:
         lat = enodeb.get_parameter(ParameterName.GPS_LAT)
         lon = enodeb.get_parameter(ParameterName.GPS_LONG)
+        ait = enodeb.get_parameter(ParameterName.GPS_AITI)
 
-        if lat != _gps_lat_cached or lon != _gps_lon_cached:
-            _cache_new_gps_coords(lat, lon)
-        return lat, lon
+        if lat != _gps_lat_cached or lon != _gps_lon_cached or lon != _gps_ait_cached:
+            _cache_new_gps_coords(lat, lon, ait)
+        return lat, lon, ait
     except (KeyError, ConfigurationError):
         return _get_cached_gps_coords()
     except ValueError:
-        logger.warning('GPS lat/long not understood (%s/%s)', lat, lon)
-        return '0', '0'
+        logger.warning('GPS lat/long/ait not understood (%s/%s/%s)', lat, lon, ait)
+        return '0', '0', '0'
 
 
-def _get_cached_gps_coords() -> Tuple[str, str]:
+def _get_cached_gps_coords() -> Tuple[str, str, str]:
```

Comment:
same here, let's define a NamedTuple called `GPSTuple` and use that

Reply:
![image](https://user-images.githubusercontent.com/87301024/147812207-df755d58-3cb2-414f-9d7d-6de3b34d43cc.png)
modified,thank you!